test/cli/lookup_test.py

```diff
@@ -897,4 +927,76 @@ def test_config_invalid(tmpdir):
         'cppcheck: error: could not load cppcheck.cfg - not a valid JSON - syntax error at line 1 near: '
     ]
 
-# TODO: test with FILESDIR
\ No newline at end of file
+# TODO: test with FILESDIR
+
+@pytest.mark.parametrize("type,file", [("addon", "misra.py"), ("config", "cppcheck.cfg"), ("library", "gnu.cfg"), ("platform", "avr8.xml")])
+def test_lookup_path(tmpdir, type, file):
```

Comment:
I asked you not to do this. And the readability of this test is beyond horrible. It is impossible to understand what this is testing. And there are no failing tests at all. The expected output is supposed to be identical for all types and that is not reflected at it. This is not usable for test-driven development of my future changes. I will post a PR as soon as I cauught up again.

Reply:
ok sure we can try to refactor it. I just fear that splitting it up is not ideal neither, this way we know PATH is tested for all variants. But yes I agree my test code is clumpsy..